src/code/z_actor.c

```diff
@@ -1903,11 +1912,12 @@ void func_8002FA60(PlayState* play) {
         gSaveContext.respawn[RESPAWN_MODE_TOP].pos.z = 0.0f;
     }
 
-    lightPos.x = gSaveContext.respawn[RESPAWN_MODE_TOP].pos.x;
-    lightPos.y = gSaveContext.respawn[RESPAWN_MODE_TOP].pos.y + 80.0f;
-    lightPos.z = gSaveContext.respawn[RESPAWN_MODE_TOP].pos.z;
-
+    // clang-format off
+    lightPos.x = gSaveContext.respawn[RESPAWN_MODE_TOP].pos.x; \
+    lightPos.y = gSaveContext.respawn[RESPAWN_MODE_TOP].pos.y + 80.0f; \
+    lightPos.z = gSaveContext.respawn[RESPAWN_MODE_TOP].pos.z; \
     Lights_PointNoGlowSetInfo(&D_8015BC00, lightPos.x, lightPos.y, lightPos.z, 0xFF, 0xFF, 0xFF, -1);
+    // clang-format on
```

Comment:
Sorry, could you explain this to a decomp noob like me?

Reply:
the comments stop the formatter from touching anything between "off" and "on". We only do this when its is *absolutely necessary* for matching purposes.

it looks like all of that stuff needs to be a single line to match lol. There are plenty of cases like this already, but ive never seen it required for something so long before

Reply:
Yeah I was wondering about the single line stuff, but I'll move to discord

Reply:
Technically, it just needs to be `lightPos.y = gSaveContext.respawn[RESPAWN_MODE_TOP].pos.y + 80.0f;` that has to be on the same line as `Lights_PointNoGlowSetInfo`, so `// clang-format off` could be 1 line lower, but then it doesn't look as nice/not grouped together